### PR TITLE
updated alcotest to 1.1.0

### PR DESCRIPTION
**Pull request by GeorgeBurac**
Signed-off-by: Gheorghe Burac <gheorghe.burac@citrix.com>

### CODE REVIEW

File: packages/upstream/alcotest.1.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/mirage/alcotest.git"
```
Comment (psafont) [inline packages/upstream/alcotest.1.1.0/opam:46]: Both files are missing the `url` section that states where the package is downloaded from, see https://github.com/ocaml/opam-repository/blob/master/packages/alcotest/alcotest.1.1.0/opam

Comment (GeorgeBurac) [reply packages/upstream/alcotest.1.1.0/opam:46]: Yes, apologies. I forgot to commit again.

Comment (psafont) [reply packages/upstream/alcotest.1.1.0/opam:46]: No need to apologize! Mistakes are the reason we have tests

**Comment by lippirk** [comment]
It looks like this breaks two repos:

```
#=== ERROR while compiling xapi-nbd.master ====================================#

# context              2.0.7 | linux/x86_64 | ocaml-base-compiler.4.08.1 | file:///mnt
# path                 ~/.opam/4.08/.opam-switch/build/xapi-nbd.master
# command              ~/.opam/4.08/bin/dune runtest
# exit-code            1
# env-file             ~/.opam/log/xapi-nbd-1034-6f403c.env
# output-file          ~/.opam/log/xapi-nbd-1034-6f403c.out
### output ###
# [...]
# File "lib_test/suite.ml", line 39, characters 45-53:
# 39 | let () = Alcotest.run "suite" [("Vbd_store", test_set)]
#                                                   ^^^^^^^^
# Error: This expression has type unit Alcotest_lwt.test_case list
#        but an expression was expected of type unit Alcotest.test_case list
#        Type
#          unit Alcotest_lwt.test_case =
#            string * Alcotest_lwt.speed_level * (unit -> Alcotest_lwt.return)
#        is not compatible with type
#          unit Alcotest.test_case =
#            string * Alcotest.speed_level * (unit -> unit) 
#        Type Alcotest_lwt.return = unit Lwt.t is not compatible with type unit 

#=== ERROR while compiling nbd-tool.4.0.1 =====================================#
# context              2.0.7 | linux/x86_64 | ocaml-base-compiler.4.08.1 | file:///mnt
# path                 ~/.opam/4.08/.opam-switch/build/nbd-tool.4.0.1
# command              ~/.opam/4.08/bin/dune runtest -p nbd-tool -j 71
# exit-code            1
# env-file             ~/.opam/log/nbd-tool-1034-f3aeab.env
# output-file          ~/.opam/log/nbd-tool-1034-f3aeab.out
### output ###
# Error: This expression has type string * unit Alcotest_lwt.test_case list
# [...]
#            string * Alcotest.speed_level * (unit -> Alcotest.return) 
#        Type Alcotest_lwt.return = unit Lwt.t is not compatible with type
#          Alcotest.return = unit 
#        suite alias cli_test/runtest
# Testing Nbd CLI interoperability tests.
# This run has ID `EED9A82C-530D-4A94-9568-641DFAA0E268`.
[OK]                NBD CLI interoperability tests          0   data copying with qemu-img.
[OK]                NBD CLI interoperability tests          1   listing exports with nbd-client.
[OK]                Stress tests                            0   Misbehaving clients sending random data.
# The full test results are available in `/home/opam/.opam/4.08/.opam-switch/build/nbd-tool.4.0.1/_build/default/cli_test/_build/_tests/EED9A82C-530D-4A94-9568-641DFAA0E268`.
# Test Successful in 0.000s. 3 tests run.
<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+- The following actions failed
| - build nbd-tool 4.0.1
| - build xapi-nbd master
```

I will fix them

**Comment by psafont** [comment]
Those seem affected by a breaking change in [1.0.0](https://github.com/mirage/alcotest/releases/tag/1.0.0):

> Change Alcotest_{async,lwt}.test_case to return monadic values. These must
be run with the new Alcotest_{async,lwt}.run functions. See
examples/lwt/test.ml for an example of the new API. (#167, @CraigFe)

**Pull request merged**